source/_integrations/sensor.rest.markdown

```diff
@@ -115,6 +115,11 @@ resource_template:
   description: The resource or endpoint that contains the value with template support.
   required: true
   type: template
+scan_interval: 
```

Comment:
This is not an option of the rest integration, but a configuration option of the platform configuration. It is documented here: <https://www.home-assistant.io/docs/configuration/platform_options/#scan-interval>